crowdsourcing/crowdsourcing_tutorial.py

```diff
@@ -1,9 +1,9 @@
 # %% [markdown]
 # # Crowdsourcing tutorial
-# In this tutorial, we'll provide a simple walkthrough of how to use Snorkel to resolve conflicts
-# in a noisy, hybrid dataset for a sentiment analysis task.
-# We have crowdsourced labels for a portion of the training dataset, and combine these
-# with heuristic labeling functions to increase the number of training labels we have.
+# In this tutorial, we'll provide a simple walkthrough of how to use Snorkel alongside Crowdsourcing to generate labels for a sentiment analysis task.
```

Comment:
nit: I don't think we should capitalize Crowdsourcing, because it's a common technique